document-models/real-world-assets/scripts/createDocumentFromJsonFiles.ts

```diff
@@ -0,0 +1,19 @@
+// import { utils } from '..';
```

Comment:
Was this file meant to be included?

Reply:
yeah I expect that I will need it again, but I commented it out because it has typescript issues that make the build checks fail.